### ISSUE TITLE
deprecate / remove -java-home on lunix

### ISSUE THREAD

**Issue by fommil** [sbt/sbt-launcher-package#114]
ticket to discuss timing of removal of `-java-home` because http://www.jenv.be/ does a much better job

https://github.com/sbt/sbt-launcher-package/pull/113

**Comment by dwijnand** [comment]
I agree that jenv does a better job, but I don't see why it shouldn't be convenient to do it outside of using or not using jenv.
